src/Domain.LinnApps/Parts/ImportBookInvoiceDetailKey.cs

```diff
@@ -5,5 +5,11 @@ public class ImportBookInvoiceDetailKey
         public int ImportBookId { get; set; }
 
         public int LineNumber { get; set; }
+
+        public ImportBookInvoiceDetailKey(int importBookId, int lineNumber)
```

Comment:
constructor should be at the top. Maybe check that your style cop plug in still activated. sometimes gets switched off when updating etc.

Reply:
yeah somethings gone wrong and format file hasn't been working like it did + stylecop seems weird, will need to fix

Reply:
check your resharper extension manager